core/src/main/java/org/mskcc/cbio/portal/dao/DaoMutation.java

```diff
@@ -1548,7 +1556,7 @@ public static void deleteAllRecords() throws DaoException {
                 try {
                     con = JdbcUtil.getDbConnection(DaoMutation.class);
                     pstmt = con.prepareStatement(
-                            "SELECT DISTINCT DRIVER_TIERS_FILTER FROM mutation "
+                            "SELECT DISTINCT DRIVER_TIERS_FILTER FROM alteration_driver_annotation "
```

Comment:
@pvannierop @dippindots this DAO only has to do with import right?  This is not related to reading db, right?